models/Adult.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const adultSchema = new Schema({
+    _id: {type: Schema.Types.ObjectId},
+    type: {type: String, required:true},
+    num: {type: Number, required:true},
+    numFledges: {type: Number, default: 0 },
+    numChicks: {type: Number, default: 0 },
+    time: {type: Date, required:true },
+    mapNumber: {type: Number, default: null},
+    gps: {type:String, required:true}, //unsure if it's required
```

Comment:
🚫 **[eslint]** <[spaced-comment](https://eslint.org/docs/rules/spaced-comment)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected exception block, space or tab after '//' in comment.
```suggestion
    gps: {type:String, required:true}, // unsure if it's required
```